app.js

```diff
@@ -1,103 +1,118 @@
-  } catch(e) {
-    res.status(500).json({e})
+  } catch (e) {
+    res.status(500).json({e});
   }
-})
+});
 
-app.get('/api/v1/cardgames', async (req, res) => {
+app.get("/api/v1/cardgames", async (req, res) => {
   try {
-    const cardGames = await database('card_games').select();
+    const cardGames = await database("card_games").select();
     res.status(200).json(cardGames);
-  } catch(e) {
-    res.status(500).json({e})
+  } catch (e) {
+    res.status(500).json({e});
   }
-})
+});
 
-app.get('/api/v1/music', async (req, res) => {
+app.get("/api/v1/music", async (req, res) => {
   try {
-    const music = await database('music').select();
+    const music = await database("music").select();
     res.status(200).json(music);
-  } catch(e) {
-    res.status(500).json({e})
+  } catch (e) {
+    res.status(500).json({e});
   }
-})
+});
 
-app.get('/api/v1/music/:genre', async (req, res) => {
+app.get("/api/v1/music/:genre", async (req, res) => {
   try {
-    const music = await database('music').where('genre', `${req.params.genre}`).select();
+    const music = await database("music")
+      .where("genre", `${req.params.genre}`)
+      .select();
     if (music.length) {
       res.status(200).json(music);
     } else {
       res.status(404).json({
-        error: `No music found with a genre of ${ req.params.genre }`
-      })
+        error: `No music found with a genre of ${req.params.genre}`,
+      });
     }
-  } catch(e) {
-    res.status(500).json({e})
+  } catch (e) {
+    res.status(500).json({e});
   }
-})
+});
 
-app.get('/api/v1/podcasts', async (req, res) => {
+app.get("/api/v1/podcasts", async (req, res) => {
   try {
-    const podcasts = await database('podcasts').select();
+    const podcasts = await database("podcasts").select();
     res.status(200).json(podcasts);
-  } catch(e) {
-    res.status(500).json({e})
+  } catch (e) {
+    res.status(500).json({e});
   }
-})
+});
 
-app.get('/api/v1/podcasts/:genre', async (req, res) => {
+app.get("/api/v1/podcasts/:genre", async (req, res) => {
   try {
-    const podcasts = await database('podcasts').where('genre', 'ilike', `%${req.params.genre}%`).select();
+    const podcasts = await database("podcasts")
+      .where("genre", "ilike", `%${req.params.genre}%`)
+      .select();
     if (podcasts.length) {
       res.status(200).json(podcasts);
     } else {
       res.status(404).json({
-        error: `No podcasts found with a genre of ${ req.params.genre }`
-      })
+        error: `No podcasts found with a genre of ${req.params.genre}`,
+      });
     }
-  } catch(e) {
-    res.status(500).json({e})
+  } catch (e) {
+    res.status(500).json({e});
   }
-})
+});
 
-module.exports = app;
\ No newline at end of file
+app.get("/api/v1/questions", async (req, res) => {
+  try {
+    const questions = await database("questions").select();
+    res.status(200).json(questions);
+  } catch (e) {
+    res.status(500).json({e});
+  }
+});
+
+module.exports = app;
```

Comment:
Thanks for removing the unnecessary spacing, and also, should we make use of double-quotes standard from now?